src/pages/Detail/Components/HappyCenter.js

```diff
@@ -55,100 +55,110 @@ class HappyCenter extends Component {
               </div>
             </li>
           </ul>
-
-          <div className="happyUnfold fst" onClick={this.unfoldClose}>
+          <div className="happyUnfold fst">
             <div className="fstLeft">
               <h4 className="tit">교환 및 환불안내</h4>
               <p className="sub">
                 교환 및 환불이 필요하신 경우 고객행복센터로 문의해주세요.
               </p>
             </div>
             <div className="fstRight">
-              <a className="asked" href="/">
-                <span className="txt">닫기</span>
-                <i className="fas fa-chevron-up" />
-              </a>
+              <button className="asked" onClick={this.unfoldBtn}>
+                {this.state.isUnfold ? (
+                  <span className="foldBtnTxt">닫기</span>
+                ) : (
+                  <span className="foldBtnTxt">자세히 보기</span>
+                )}
```

Comment:
필요한 부분만 삼항연산자 적용하시면 훨씬 깔끔할 것 같네요.
구조분해할당 하시면 더 좋을 것 같습니다.

바로 아래 코드도 리뷰 반영해서 적용해주세요!

```suggestion
       <span className="foldBtnTxt">{this.state.isUnfold ? "닫기" : "자세히 보기"}</span>
```